src/validate.js

```diff
@@ -2964,6 +2964,46 @@ function F_VALIDATE(disabledHL) {
 					segment.report(154);
 			} // GROUP isDrivable && Primary
 
+			// GROUP isDrivable && Local Street
+			if (RT_RAILROAD === roadType) {
```

Comment:
I believe RT_RAILROAD is a typo here.

Reply:
Ooops, that's what happens when you copy/paste, and forget to check in fixes after the fact.  Committed changes for both issues.